route.go

```diff
@@ -39,6 +39,7 @@ type Route struct {
 	buildVarsFunc BuildVarsFunc
 }
 
+// SkipClean returns whether accessing "/path//to" will not redirect
```

Comment:
The phrasing of this is kind of a double negative. I'd suggest documenting it similarly to Router.SkipClean